uaclient/snap.py

```diff
@@ -93,3 +95,44 @@ def get_config_option_value(key: str) -> Optional[str]:
         return out.strip()
     except exceptions.ProcessExecutionError:
         return None
+
+
+def get_snap_package_info_tracking(package: str) -> Optional[str]:
+    out, _ = system.subp(["snap", "info", package])
+    match = re.search(r"tracking:\s+(?P<tracking>.*)", out)
+    if match:
+        return match.group("tracking")
+    return None
+
+
+class SnapPackage:
+    def __init__(self, name, version, rev, tracking, publisher, notes):
+        self.name = name
+        self.version = version
+        self.rev = rev
+        self.tracking = tracking
+        self.publisher = publisher
+        self.notes = notes
+
+
+def get_installed_packages() -> List[SnapPackage]:
+    out, _ = system.subp(["snap", "list"])
```

Comment:
The output of that call could have `unicode`  and `color`values. I think it is safe for us to use `--color never --unicode never` here